src/target/source/codegen_c_host.cc

```diff
@@ -386,7 +393,7 @@ runtime::Module BuildCHost(IRModule mod, Target target) {
     // symbols are available to tvm_run_func
     auto fun_name = std::string(kv.first->name_hint);
     const bool is_aot_executor_fn =
-        (fun_name.rfind(::tvm::runtime::symbol::tvm_run_func_prefix, 0) == 0);
+        (fun_name.rfind(::tvm::runtime::symbol::tvm_run_func_suffix) != std::string::npos);
```

Comment:
Do you mean adding an attribute to the TIR function, e.g., `runner_func` and then this would become 
`const bool is_aot_executor_fn = (kv.first->GetAttr("runner_func") == true)`?

Reply:
yes, although would suggest to include "aot_" in the attr name and create a constant